### DISCUSSION TITLE
[feature] Feeds with a filter option.

### DISCUSSION THREAD

**Discussion by mrprajesh** [community/community#15318] [Other Feature Feedback, Questions, & Ideas]
Currently, the feed appears like `all in one` -- the people we follow and the repo we watch. Suppose we have option to `tag` people/repo while clicking `follow` or `watch`. Then we can group or filter to choose it on my feed.

**use case:**
1. Suppose I have `cpp` members and repo I follow and similarly some other language. say `cuda` members and repo's I follow. Having a filter to choose group any of them will greatly benefit the feed options.

**Comment by siminapasat** [comment]
Oh interesting, thank you @mrprajesh for your feedback. We're currently working on filtering based on content type, but differentiating per people / repo is interesting 🤔 .

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
